lib/writeTempConfig.js

```diff
@@ -18,16 +19,24 @@ process.on('exit', function () {
   }
 });
 
-module.exports = function (config, esPath) {
+module.exports = function (config, logLevel, esPath) {
   return mkdirTemp({prefix: 'libesvm-'}).then(function createConfig(dir) {
-    var configFileDestination = path.join(dir, 'elasticsearch.json');
-
-    return Promise.all([
-      writeFile(configFileDestination, JSON.stringify(config), 'utf8'),
-      copy(join(esPath, 'config'), dir)
-    ])
+    return Promise.attempt(function () {
```

Comment:
> For compatibility with earlier ECMAScript version, an alias Promise.attempt is provided for Promise.try.

I don't know the ins and outs of all the ECMAScript/A+ Promise rules, but this sounds like `attempt` is old or possibly deprecated, maybe in favor of `try`, no?